frontend/device/remarkable/event_map.lua

```diff
@@ -1,7 +1,4 @@
 return {
-    [102] = "Home",
-    [105] = "LPgBack",
-    [106] = "RPgFwd",
```

Comment:
These are safe to keep. It just maps event codes that will never be fired on a RM2